v0.1/elements/image.schema.json

```diff
@@ -31,9 +33,18 @@
 	{"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE"}
       ]
     },
-    "indexingParameters": {      
-      "definition": "Indexing parameters that apply to the image.",
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/INDEXING_PARAMETERS"
-    }	  
+    "unit": {
+      "description": "A determinate quantity adopted as a standard of the measurement.",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM"
+    },    
+    "voxelSize": {
+      "type": "array",
+      "description": "Size of one voxel (3D) or pixel (2D, with z = 0).",
```

Comment:
Maybe we should mention that the voxelSize is in relation to the unit.
For example: "Size of one voxel (3D) or pixel (2D, with z = 0), in the unit defined for this image"

Reply:
I like that suggestion. I'll quickly add it :)

Reply:
We could also specify that it is the size of the voxel in the physical space, that would be as specific as possible.

Reply:
> Corresponding size of one voxel (3D) or pixel (2D, with z = 0) in the physical space, defined in the unit of this image.

Reply:
Even better! Changed the description.